docs/sources/developers/plugins/publish-a-plugin.md

```diff
@@ -7,140 +7,39 @@ title: Publish a plugin
--rw-r--r--      39406 bX defN 22-Mar-24 23:23 grafana-clickhouse-datasource/dashboards/query-analysis.json
--rwxr-xr-x   16469136 bX defN 22-Mar-24 23:23 grafana-clickhouse-datasource/gpx_clickhouse_darwin_amd64
--rwxr-xr-x   16397666 bX defN 22-Mar-24 23:23 grafana-clickhouse-datasource/gpx_clickhouse_darwin_arm64
--rwxr-xr-x   14942208 bX defN 22-Mar-24 23:23 grafana-clickhouse-datasource/gpx_clickhouse_linux_amd64
--rwxr-xr-x   14155776 bX defN 22-Mar-24 23:23 grafana-clickhouse-datasource/gpx_clickhouse_linux_arm
--rwxr-xr-x   14548992 bX defN 22-Mar-24 23:23 grafana-clickhouse-datasource/gpx_clickhouse_linux_arm64
--rwxr-xr-x   15209472 bX defN 22-Mar-24 23:23 grafana-clickhouse-datasource/gpx_clickhouse_windows_amd64.exe
-drwxr-xr-x          0 bx stor 22-Mar-24 23:23 grafana-clickhouse-datasource/img/
--rw-r--r--        304 bX defN 22-Mar-24 23:23 grafana-clickhouse-datasource/img/logo.png
--rw-r--r--       1587 bX defN 22-Mar-24 23:23 grafana-clickhouse-datasource/img/logo.svg
--rw-r--r--     138400 bX defN 22-Mar-24 23:23 grafana-clickhouse-datasource/module.js
--rw-r--r--        808 bX defN 22-Mar-24 23:23 grafana-clickhouse-datasource/module.js.LICENSE.txt
--rw-r--r--     487395 bX defN 22-Mar-24 23:23 grafana-clickhouse-datasource/module.js.map
--rw-r--r--       1616 bX defN 22-Mar-24 23:23 grafana-clickhouse-datasource/plugin.json
-22 files, 92516655 bytes uncompressed, 34319591 bytes compressed:  62.9%
-```
-
-## Publish your plugin on Grafana.com
-
-The best way to share your plugin with the world is to publish it on [Grafana Plugins](/plugins).
-By having your plugin published on Grafana.com, more users will be able to discover your plugin.
-
-Before you submit your plugin, we ask that you read our guidelines and frequently asked questions.
-
-### Guidelines
-
-To speed up the time it takes to review your plugin:
-
-- Get familiar with our plugin [publishing and signing criteria]({{< relref "./publishing-and-signing-criteria" >}})
-- Check that your plugin is ready for review using the [plugin validator](https://github.com/grafana/plugin-validator).
-- Read our [6 tips for improving your Grafana plugin before you publish](/blog/2021/01/21/6-tips-for-improving-your-grafana-plugin-before-you-publish/).
-- Refer to [plugin-examples](https://github.com/grafana/grafana-plugin-examples) to review best practices for building your plugin.
-
-### Frequently Asked Questions
-
-**Do I need to submit a private plugin?**
-
-- No. Please only submit plugins that you wish to make publicly available for the Grafana community.
-
-**How long does it take to review my submission?**
-
-- We're not able to give an estimate at this time, though we're constantly working on improving the time it takes to review a plugin.
+# Publish or update a plugin
 
-**Can I decide a date when my plugin will be published?**
+You've just built your first plugin and [packaged it](package-a-plugin.md), and now you want to share it with the world.
```

Comment:
this link 404s for me on a local build as it tries to get to http://localhost:3002/docs/grafana/latest/developers/plugins/publish-a-plugin/package-a-plugin.md  rather than the correct - http://localhost:3002/docs/grafana/latest/developers/plugins/package-a-plugin/

Reply:
As in the previous PR, I would like to remove notions of this being your first plugin.

Reply:
@jdbaldry While fixing a relref in the doc, I identified three places where a "Sign in" link to Grafana Cloud was given with a relative path and no relref. I fixed it, but want to check to see if there's something special about Cloud's sign in that would make it an exception?

Reply:
The destinations in https://github.com/grafana/grafana/pull/67164/commits/663481ba34bf0afa7db8b371e40d3a08d4ef6846#diff-0d9267ff4b6486865c95041d2198385726213f3fd9b8736900104c49ca8359a8L42 are actually absolute URL paths because of the preceding slash (`/`). A relative path would start with one or two periods (`./` or `../`).

I realized the updated documentation doesn't clearly explain when to use relrefs, so I added a new note: https://github.com/grafana/grafana/pull/67164/commits/663481ba34bf0afa7db8b371e40d3a08d4ef6846#diff-0d9267ff4b6486865c95041d2198385726213f3fd9b8736900104c49ca8359a8L42.

In this case, because the link is external to the project documentation, it should not be a relref.

Reply:
Ok, will check in fix.